core/src/main/java/com/netflix/iceberg/MergingSnapshotUpdate.java

```diff
@@ -436,7 +436,7 @@ private ManifestFile filterManifest(Expression deleteExpression,
     // use a lookback of 1 to avoid reordering the manifests. using 1 also means this should pack
     // from the end so that the manifest that gets under-filled is the first one, which will be
     // merged the next time.
-    ListPacker<ManifestFile> packer = new ListPacker<>(manifestTargetSizeBytes, 1);
+    ListPacker<ManifestFile> packer = new ListPacker<>(manifestTargetSizeBytes, 1, false);
```

Comment:
Callout : disabled by the flag to pick expensive task first to mirror default value. Although tests pass, please advise if setting this `false` here is the right way to go? cc: @rdblue

Reply:
`false` is the correct setting.

Iceberg tries to avoid reordering manifests so that the manifests and data files are roughly kept in order of most to least recently modified.

Reply:
+1 thank you for the explanation.